deployment/build_and_release/modules/release/variables.tf

```diff
@@ -14,8 +15,25 @@ variable "bucket_env" {
 
 variable "signing_keyring_location" {
   description = "The GCP location to create the signing keyring"
+  type        = string
 }
 
 variable "tf_state_location" {
   description = "The GCP location to store Terraform remote state"
+  type        = string
+}
+
+variable "cloudbuild_path" {
+  description = "The path of the Cloud Build config in the Github repo"
```

Comment:
Which github repo is this talking about? At the moment this is just doing applet, but what happens with OS and boot?